src/models/odahuflow/ModelTrainingSpec.ts

```diff
@@ -33,7 +38,7 @@ export interface ModelTrainingSpec {
     /**
      * Model training hyperParameters in parameter:value format
      */
-    hyperParameters?: any;
+    hyperParameters?: { [key: string]: string; };
```

Comment:
This is the reason of failing build. Not sure why it happens, cause I don't know TypeScript, but I have faced this before.